docs/src/main/asciidoc/elasticsearch-dev-services.adoc

```diff
@@ -48,14 +49,20 @@ Note that the Elasticsearch hosts property is automatically configured with the
 
 == Configuring the image
 
-Dev Services for Elasticsearch only support Elasticsearch based images, OpenSearch is not supported at the moment.
+Dev Services for Elasticsearch support distributions based on both Elasticsearch and OpenSearch images.
```

Comment:
```suggestion
Dev Services for Elasticsearch support distributions based on both Elasticsearch and OpenSearch images.

When using Hibernate Search, Dev Services will default to Elasticsearch or OpenSearch based on Hibernate Search configuration.

Otherwise, Dev Services will default to Elasticsearch. To use OpenSearch, configure the distribution explicitly:
[source,properties,subs="attributes"]
----
quarkus.elasticsearch.devservices.distribution=opensearch
----

```